x-pack/plugins/lens/public/indexpattern_datasource/field_item.tsx

```diff
@@ -543,3 +607,34 @@ function FieldItemPopoverContents(props: State & FieldItemProps) {
   }
   return <></>;
 }
+
+const DragToWorkspaceButton = ({
+  field,
+  dropOntoWorkspace,
+  isEnabled,
+}: {
+  field: Dragging;
+  dropOntoWorkspace: DatasourceDataPanelProps['dropOntoWorkspace'];
+  isEnabled: boolean;
+}) => {
+  const buttonTitle = i18n.translate('xpack.lens.indexPattern.moveToWorkspace', {
+    defaultMessage: 'Add field to workspace',
+  });
+
+  if (isEnabled) {
+    return (
+      <EuiFlexItem grow={false}>
+        <EuiButtonIcon
+          aria-label={buttonTitle}
+          iconType="plusInCircle"
+          onClick={() => {
+            dropOntoWorkspace(field);
+          }}
+          title={buttonTitle}
+        />
+      </EuiFlexItem>
```

Comment:
I think I would have appreciated an EuiTooltip here instead of a `title` property- it seems like this button is potentially confusing for mouse users.